java/src/main/kotlin/org/digma/intellij/plugin/idea/psi/java/JavaAnnotation.kt

```diff
@@ -0,0 +1,10 @@
+package org.digma.intellij.plugin.idea.psi.java
+
+import com.intellij.psi.PsiClass
+
+data class JavaAnnotation(
+    val fqn: String,
```

Comment:
what means 'fqn'? maybe you can provide here better name for this variable so it will be clear for other devs?

Reply:
its class name FQN, i will rename it